### PR TITLE
Let ‘button of save’ & ‘button of cancel’ have a opportunity  to be used -- Advanced Form

### PR DESCRIPTION
**Pull request by wunayou**
It  saves  the  input content  when  lost  focus.  So,  There is no opportunity  for ‘button of save’ & ‘button of cancel’  to be used.

### CODE REVIEW

File: src/routes/Forms/TableForm.js [hunk 1]
```diff
@@ -23,6 +23,17 @@ export default class TableForm extends PureComponent {
...
+  //   e.preventDefault();
+  //   this.props.form.validateFieldsAndScroll((err, values) => {
+  //     if (!err) {
+  //       this.props.dispatch({
+  //         type: 'form/submit',
+  //         payload: values,
+  //       });
+  //     }
+  //   });
+  // }
```
Comment (ddcat1115) [inline src/routes/Forms/TableForm.js]: 这个现在没用，直接删掉好了

**Comment by chenshuai2144** [comment]
blur也蛮好用的。。好像点击了一保存一下

**Comment by ddcat1115** [comment]
blur 保存会引发一些比较烦的问题，这样改下比较好

**Comment by ddcat1115** [comment]
#846

**Pull request merged**